docs/docs/backend/Server.md

```
@@ -1986,6 +2030,28 @@ class Query(graphene.ObjectType)
 
 Define GraphQL queries.
 
+<a id="db.schemas.Query.resolve_deviceMetrics"></a>
+
+#### resolve\_deviceMetrics
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)